mitmproxy/optmanager.py

```diff
@@ -541,31 +541,38 @@ def parse(text):
     return data
 
 
-def load(opts: OptManager, text: str) -> None:
+def load(opts: OptManager, text: str, cwd: Path | str | None = None) -> None:
     """
     Load configuration from text, over-writing options already set in
     this object. May raise OptionsError if the config file is invalid.
     """
     data = parse(text)
+
+    scripts = data.get("scripts")
+    if scripts is not None and cwd is not None:
+        data["scripts"] = [
+            str(relative_path(Path(path), relative_to=Path(cwd))) for path in scripts
```

Comment:
If cwd is None, `Path(cwd)` will fail I think?
(reviewing from mobile and can't verify)

Reply:
Yes, but we check that it is not None on [line 552](https://github.com/mitmproxy/mitmproxy/pull/4860/files#diff-6b02b884786fb205ec197276b4fc6434ff3d51f37fea73479f8c71a888565b6aR552).